test_environment-v5.yml

```diff
@@ -0,0 +1,16 @@
+name: test-environment-v5
+channels:
+- conda-forge
+dependencies:
+- python==3.11
```

Comment:
Why not the latest version? (I feel like it would make sense to just always test with the latest python, numpy, scipy, matplotlib?)